DOCS-821: Add Power Sensor Component (#1766)
viamrobotics/docs: docs/components/power-sensor/renogy.md

---

> title: "Configure a renogy power sensor"

Reviewer: ```suggestion
title: "Configure a renogy Power Sensor"
```

linkTitle: "renogy"
weight: 10
type: "docs"
description: "Configure a renogy model power sensor to return battery temperature."
tags: ["power sensor", "components", "renogy"]
icon: "/icons/components/sensor.svg"
images: ["/icons/components/sensor.svg"]
# SME: #team-bucket
---

Configure a `renogy` sensor to integrate a [Renogy battery temperature sensor](https://www.renogy.com/battery-temperature-sensor-for-renogy-solar-charge-controllers/) into your robot:

> {{< tabs name="Configure an Renogy Sensor" >}}

Reviewer: ```suggestion
{{< tabs name="Configure a Renogy Power Sensor" >}}
```

# SME: #team-bucket
---

> Configure a `renogy` sensor to integrate a [Renogy battery temperature sensor](https://www.renogy.com/battery-temperature-sensor-for-renogy-solar-charge-controllers/) into your robot's system:

Reviewer: ```suggestion
Configure a `renogy` sensor to integrate a [Renogy battery temperature sensor](https://www.renogy.com/battery-temperature-sensor-for-renogy-solar-charge-controllers/) into your robot:
```

For this, I know "into your robot" is a little awkward. We're in the process of changing the language around this, so for now let's use the same language we use elsewhere (also makes the future find and replace easier).

Love the link change!

{{< tabs name="Configure an Renogy Sensor" >}}
{{% tab name="Config Builder" %}}

Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).

> Click on the **Components** subtab and navigate to the **Create component** menu. Select the type `power_sensor`, then select the `renogy` model. Name your sensor, and click **Create**.

Reviewer: ```suggestion
Click on the **Components** subtab and navigate to the **Create component** menu.
Select the type `power_sensor`, then select the `renogy` model.
Name your sensor, and click **Create**.
```
